src/jit/importer.cpp

```diff
@@ -4693,6 +4693,107 @@ void           Compiler::impImportAndPushBox (CORINFO_RESOLVED_TOKEN * pResolved
     impPushOnStack(op1, tiRetVal);
 }
 
+/*****************************************************************************
+*
+*  Build and import `new` of multi-dimmensional array
+*/
+
+void           Compiler::impImportNewObjArray(CORINFO_RESOLVED_TOKEN* pResolvedToken,
```

Comment:
The description should include how the args are passed (arg count, followed by the block of N integer args.